src/SFA.DAS.PR.Jobs/Models/Recruit/ResponseBase.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace SFA.DAS.PR.Jobs.Models.Recruit;
+
+public enum ResponseCode
+{
+    Success,
+    InvalidRequest,
+    NotFound,
+    Created
+}
+
+public class DetailedValidationError
```

Comment:
We could have avoided this, just need to look at response code and react accordingly.